tests/translator/input/api_with_resource_policy_global.yaml

```diff
@@ -0,0 +1,36 @@
+Globals:
+  Api:
+    Auth:
+      ResourcePolicy:
+        CustomStatements: [{
+            Action: 'execute-api:Invoke', 
+            Resource: ['execute-api:/*/*/*']
+          }]
+
+Resources:
+ ExplicitApi:
+    Type: AWS::Serverless::Api
+    Properties:
+      StageName: Prod
+      DefinitionBody: {
+        "info": {
+          "version": "1.0",
+          "title": {
+            "Ref": "AWS::StackName"
+          }
+        },
+        "paths": {},
+        "swagger": "2.0",
+        "x-amazon-apigateway-policy": {
```

Comment:
I'm having trouble finding the API GW docs on this OpenApi extension. Could you point me to them? Just want to make sure they're in the right place in the overall swagger doc.

Reply:
I don't see it in the OpenApi extensions: https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions.html

Reply:
I got this from API gateway, by manually updating the API with resource policies on the apigw console and exporting the swagger with the apigw extensions.

Reply:
You should contact them to update their docs to add this extension.